lib/enkf/ecl_config.cpp

```diff
@@ -133,13 +124,17 @@ ui_return_type * ecl_config_validate_data_file(const ecl_config_type * ecl_confi
   }
 }
 
+static char *alloc_tagged_string(const char *s) {
+  return util_alloc_sprintf("%s%s%s", DEFAULT_START_TAG, s, DEFAULT_END_TAG);
+}
+
 
 void ecl_config_set_data_file(ecl_config_type * ecl_config, const char * data_file) {
   ecl_config->data_file = util_realloc_string_copy(ecl_config->data_file, data_file);
   {
     FILE * stream = util_fopen(ecl_config->data_file, "r");
     basic_parser_type * parser = basic_parser_alloc(NULL, NULL, NULL, NULL, "--", "\n");
-    char * init_tag = enkf_util_alloc_tagged_string("INIT");
+    char * init_tag = alloc_tagged_string("INIT");
```

Comment:
Consider inlining as `alloc_tagged_string` occurs only once.

```c++
const char * init_tag = DEFAULT_START_TAG "INIT" DEFAULT_END_TAG;
/* Or better yet: */
auto init_tag = "<INIT>";
```
and remove the `free(init_tag);`